<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformanceDialog</class>
 <widget class="QDialog" name="PerformanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Performance Mode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog 
{ 
	background:  rgba(32, 80, 96, 100);
	/*background:  rgba(255, 0, 0, 255);*/
	color:white;
	font-size: 12px;
}

QPushButton
{
	color: white;
	background-color: #27a9e3;
	border-width: 0px;
	border-radius: 3px;
}

QLabel , QRadioButton, QCheckBox
{ 
	color: white; 
	border-radius: 3px;
}

QPushButton:hover, QComboBox:hover{ background-color: #66c011; }
QPushButton:disabled, QComboBox:disabled { background-color:rgb(114,200,237); }

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="3" column="0">
    <widget class="QPushButton" name="safeStopBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 200, 0)</string>
     </property>
     <property name="text">
      <string>Safe Stop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="leftBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Left</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="forwardBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Forward</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="stopBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="rightBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="backwardBtn">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>微軟正黑體</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Backward</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>5</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QPushButton" name="audioBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Audio Action</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="armBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Arm Action</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="ledBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LED Action</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="audioCb">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Meals ready</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thank you</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Be careful with hot meals</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Be careful with robot</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Background music ON</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Background music OFF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Happy Birthday</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Welcome</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thanks for coming</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio 9</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="ledCb">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Hello</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Delivery</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Please take meal</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Thanks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Empty</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="deliveryBtn">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Delivery Action</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QComboBox" name="tableCb">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QComboBox" name="armCb">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>24</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>Extension</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Retraction</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="abortBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Termination</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goOriginBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Go Origin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetOriginBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Origin Positioning</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
